node/ledger/src/lib.rs

```diff
@@ -136,12 +150,16 @@ impl<N: Network, C: ConsensusStorage<N>> Ledger<N, C> {
         ledger.current_block = Arc::new(RwLock::new(block));
         // Set the current epoch challenge.
         ledger.current_epoch_challenge = Arc::new(RwLock::new(Some(ledger.get_epoch_challenge(latest_height)?)));
+        lap!(timer, "Initialize ledger state");
+
+        // Safety check the existence of every block.
+        cfg_into_iter!((0..=latest_height)).try_for_each(|height| {
```

Comment:
Actually, could we do `.chunks(50)` (or 100+) and call `ledger.get_blocks(start, end)`?

Reply:
Should we add `cfg_iter` to `Ledger::get_blocks` too? IIRC it's just a linear scan of a given range at the moment.